addons/udes_stock/models/stock_picking.py

```diff
@@ -267,6 +267,21 @@ def _compute_num_pallets(self):
         for picking in self:
             picking.u_num_pallets = len(picking.move_line_ids.result_package_id)
 
+    def reset_move_lines_from_pack(self):
+        """Resets move lines from location destination, quantity done, completed 
+         datetime and done status in the packing process to default or Null values
+         from self recordset
+        """
+        done_move_lines = self.move_line_ids.filtered(
+            lambda ml: ml.location_dest_id 
+            and ml.qty_done 
+            and ml.u_done_datetime
+            and ml.u_done_by_id
+        )
+        done_move_lines.reset_move_line_data(self.picking_type_id.default_location_dest_id.id)
```

Comment:
Question:

Do we need to pass the default_location_dest_id in this method as parameter, i think is not needed.
In reset_move_line_data can say:
default_location_dest = self.u_picking_type_id.default_location_dest_id